index.html

```diff
@@ -300,6 +321,12 @@
             applications progressively render to a canvas, this allows
             applications to avoid capturing a partially rendered frame.
           </p>
+          <p class="note">
+            As currently specified, this results in
+            no <code>SecurityError</code> or other error feedback if the canvas
+            is not origin-clean.  In part, this is because we don't track where
```

Comment:
Hmm, are these double-spaces some magic syntax? Because they're all over the place :-)